lib/matplotlib/projections/polar.py

```diff
@@ -987,12 +987,26 @@ def set_thetalim(self, *args, **kwargs):
         where minval and maxval are the minimum and maximum limits. Values are
         wrapped in to the range :math:`[0, 2\pi]` (in radians), so for example
         it is possible to do ``set_thetalim(-np.pi / 2, np.pi / 2)`` to have
-        an axes symmetric around 0.
+        an axes symmetric around 0. If not within range :math:`[0, 2\pi]` a
+        ValueError is raised.
         """
+        thetamin = 0
+        thetamax = 0
         if 'thetamin' in kwargs:
-            kwargs['xmin'] = np.deg2rad(kwargs.pop('thetamin'))
+            thetamin = np.deg2rad(kwargs.pop('thetamin'))
+            kwargs['xmin'] = thetamin
         if 'thetamax' in kwargs:
-            kwargs['xmax'] = np.deg2rad(kwargs.pop('thetamax'))
+            thetamax = np.deg2rad(kwargs.pop('thetamax'))
+            kwargs['xmax'] = thetamax
+
+        if args.__len__() == 2:
```

Comment:
In Python you do:
```suggestion
        if len(args) == 2:
```

Reply:
Yes, I should know this, will make the changes

Reply:
done!!